static/src/javascripts-legacy/projects/common/modules/experiments/ab.js

```diff
@@ -87,9 +87,12 @@ define([
     }
 
     function getActiveTests() {
-        var now = new Date();
+        var today = new Date();
+        // new Date(test.expiry) sets the expiry time to 00:00:00
+        // Using SetHours allows a test to run until the END of the expiry day
+        today.setHours(0,0,0,0);
```

Comment:
When the client currently runs the code, the value from `new Date()` is something like:
 `Tue Jan 24 2017 16:15:48`

Which would be fine, but `test.expiry` doesn't have a time, just a date, so it defaults the time to `00:00:00`. This means that the `var expired` for a test **expiring today**, would become: 
`Tue Jan 24 2017 00:00:00`

... this makes line 92 something like:

`(Tue Jan 24 2017 16:15:48) - (Tue Jan 24 2017 00:00:00)` (equals 58548041)
`58548041 > 0` (TRUE)

I'm using setHours to basically make the time of day no longer apply, so the comparisons become:
`Tues Jan 24 2017 00:00:00` > `Tues Jan 24 2017 00:00:00` (FALSE)
`Tues Jan 25 2017 00:00:00` > `Tues Jan 24 2017 00:00:00` (TRUE)

JavaScript Date/Time... yaaaaayyy~